### DISCUSSION TITLE
TypeError: unhashable type: 'numpy.ndarray' when the input is 3D data

### DISCUSSION THREAD

**Discussion by tmclouisluk** [pycaret/pycaret#3664] [Q&A]
I have a set of data with shape (n, 300, 300). However, when I use the API of 
`clf1 = setup(data=data_raw, target=y)`

It pops out the error
`TypeError: unhashable type: 'numpy.ndarray'`

Does PyCaret support 3D data?

**Comment by celestinoxp** [comment]
Can you provide more information?
- code + dataframe
- package versions
